llvm/lib/Target/AArch64/AArch64InstrInfo.cpp

```diff
@@ -2221,6 +2222,7 @@ unsigned AArch64InstrInfo::isStoreToStackSlot(const MachineInstr &MI,
   case AArch64::STRDui:
   case AArch64::STRQui:
   case AArch64::STR_PXI:
+  case AArch64::STR_PPXI:
```

Comment:
Which test does this change affect?

Given that this store takes two P registers as operands, are the operand numbers below correct? (i.e. is MI.getOperand(1).isFI() ever true for STR_PPXI?

Reply:
Hmm, very likely they are here by mistake. I'll retest and will probably remove these two changes.

Reply:
Actually, I think the changes are correct and perhaps LDR/STR_ZZXI/etc need to be added here as well.
The store takes a single register as operand (representing a register pair).
Perhaps, I'll be able to construct a test case showing how it makes a difference, say in MachineLICM or in StackSlotColoring.

Reply:
Is it required though? or does it provide extra optimisation abilities? If it's the latter then probably best done as a follow patch.

Reply:
Done.